lib/logstash/inputs/kafka.rb

```diff
@@ -155,6 +158,7 @@ def run(logstash_queue)
 
   public
   def stop
+    @kafka_client_queue.push(LogStash::SHUTDOWN)
```

Comment:
hmmm. sounds fair to keep `SHUTDOWN` only to the main logstash queue. I'll go ahead and introduce a new constant